specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-08-01/managedClusters.json

```diff
@@ -1593,10 +1593,72 @@
             "modelAsString": true
           },
           "description": "The load balancer sku for the managed cluster."
+        },
```

Comment:
adding new properties to stable version is breaking change. normally we suggested to add a new service tag.

Reply:
As these would fall into the breaking change category, please create a new API version to make these changes:
https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?e=xEcppo